pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "codemark"
+version = "0.2.1"
+description = "CodeMark CLI is a command-line interface that helps you manage your coding assignments and tests.\\nWith this tool, you can easily initialize the configuration, list assignments, fetch and check your code, submit your code for grading, and get AI-powered error recommendations."
+authors = ["rootCircle"]
+license = "Apache 2.0"
+readme = "README.md"
+
```

Comment:
Missing some important meta like github url, maintainers, keywords and other package_data as in setup.py
>    package_data={'codemark': ['firebase/res/service-account-file.json']}


https://packaging.python.org/en/latest/specifications/declaring-project-metadata/

Reply:
@rootCircle 
Package data has been added to the TOML file

Reply:
I believe other meta content like urls, and maintainers and keywords still need to added in! :-)
follow these for help : https://packaging.python.org/en/latest/specifications/declaring-project-metadata/